src/internal/pthread_impl.h

```diff
@@ -150,6 +150,8 @@ void __vm_unlock(void);
 
 int __timedwait(volatile int *, int, clockid_t, const struct timespec *, int);
 int __timedwait_cp(volatile int *, int, clockid_t, const struct timespec *, int);
+
+#ifndef __wait
```

Comment:
I had to do this in `src/thread/__wait.c` as well in order to compile successfully: https://github.com/jfbastien/musl/commit/b87ccd69c6f41f690b051794249991eb5a4267d0